nikola/data/themes/base/templates/sectionindex.tmpl

```diff
@@ -12,10 +13,8 @@
 <div class="sectionindex">
     <header>
         <h2><a href="${_link('section_index', posts[0].section_slug())}">${title|h}</a></h2>
-        % if generate_atom:
-            <p class="feedlink"><a href="${_link('section_index_atom', posts[0].section_slug())}" type="application/atom+xml">${messages('Updates')}</a></p>
-        % endif
-    </header>    
+        ${feed.link(posts[0].section_slug())}
```

Comment:
I *think* `section` will work here (and in the above `section_index` link) instead.